src/components/onboarding/Onboarding.tsx

```diff
@@ -339,6 +339,7 @@ async function createInitialAccounts(setAccounts: Function, currency: Commodity)
       placeholder: true,
       fk_commodity: currency,
       parent: root,
+      hidden: true,
```

Comment:
hide equity account as we don't use it for anything other than opening balances and may confuse users